docs/src/main/sphinx/release/release-405.md

```diff
@@ -2,10 +2,15 @@
 
 ## General
 
+* Add support for setting an optional limit for the `query_max_run_time` session
+  property, which can be set using the `query.max-run-time.hard-limit`
+  configuration property. ({issue}`15115`)
 * Improve performance of queries that compare `date` columns with
   `timestamp(n) with time zone` literals. ({issue}`5798`)
 * Improve performance and resource utilization when inserting into partitioned
   and bucketed tables. ({issue}`14718`)
+* Improve performance by removing the maximum limit of partitions which can be
```

Comment:
This sounds weird .. but I cant think of an improved version atm.

Reply:
Why would removing the maximum limit improve performance? 

Maybe just say "Improve performance for INSERT queries when fault-tolerant execution is enabled"